lib/Weight/MockWeight.hpp

```diff
@@ -0,0 +1,31 @@
+#include "IWeight.hpp"
+
+/**
+ * @brief Mock weight class that inherits from the IWeight interface class.
+ * Used for unit tests so that actual hardware isn't require to test the ZotBins
+ * system.
+ */
+
```

Comment:
Remove whitespace at line 8, so it's more clear that the comment belongs to the class